code/modules/reagents/Chemistry-Recipes.dm

```diff
@@ -1133,7 +1132,7 @@
 		/mob/living/simple_animal/hostile/mimic/crate,
 		/mob/living/simple_animal/hostile/mimic/crate/chest,
 		/mob/living/simple_animal/hostile/mimic/crate/item,
-		) + typesof(/mob/living/simple_animal/hostile/humanoid) + typesof(/mob/living/simple_animal/hostile/asteroid) //Exclusion list for things you don't want the reaction to create.
```

Comment:
Yeah, usually xenobio antags go on phazon rampages or release slimes.
It's usually a traitor miner that releases a shit load of monsters.